rules/manager/manager.go

```diff
@@ -229,9 +236,11 @@ func (m *ruleManager) runIteration(results chan<- *extraction.Result) {
 					Timestamp: s.Timestamp,
 				}
 			}
-			m.results <- &extraction.Result{
-				Samples: samples,
-				Err:     err,
+
+			if err != nil {
+				evalFailures.Inc()
```

Comment:
We could do that. `glog.V`, `glog.Warning`, or `glog.Error`? I guess one of the latter, since the logging output could be big if things go wrong, but not massive?